webapp/src/main/java/org/apache/atlas/web/rest/TypeCacheRefreshREST.java

```diff
@@ -75,20 +79,21 @@ private void refreshTypeDef(int expectedFieldKeys) throws RepositoryException, I
 
         while (currentSize != expectedFieldKeys && counter++ < totalIterationsAllowed) {
             currentSize = provider.get().getManagementSystem().getGraphIndex(VERTEX_INDEX).getFieldKeys().size();
-            LOG.info("field keys size found = {} at iteration {}", currentSize, counter);
+            LOG.info("field keys size found = {} at iteration {} :: traceId {}", currentSize, counter, traceId);
             Thread.sleep(sleepTimeInMillis);
         }
         //This condition will hold true when expected fieldKeys did not appear even after waiting for totalWaitTimeInMillis
         if (counter > totalIterationsAllowed) {
-            LOG.error("Could not find desired count of fieldKeys {} after {} ms of wait. Current size of field keys is {}", expectedFieldKeys, totalWaitTimeInMillis, currentSize);
+            LOG.error("Could not find desired count of fieldKeys {} after {} ms of wait. Current size of field keys is {} :: traceId {}", expectedFieldKeys,
```

Comment:
No need to log the error here and instead throw a more detailed exception and refreshCache method will automatically handle the endpoint

Reply:
Makes sense.

Reply:
If we don't have any other review comments then can we address this comment in the next PR?  As this branch is already merged to beta so we can merge the same to staging and master first.

Reply:
Just commit new changes to this branch only (hsexperiment-logging) and create a PR to beta again

Reply:
This is incorporated